salad/cli.py

```diff
@@ -67,10 +69,26 @@ def main(args=sys.argv[1:]):
         name += parsed_args.name
     world.remote_capabilities['name'] = name
 
-    if not parsed_args.timeout:
-        world.remote_capabilities['idle-timeout'] = 120
-    else:
+    # timeout
+    if parsed_args.timeout:
         world.remote_capabilities['idle-timeout'] = parsed_args.timeout
+    else:
+        world.remote_capabilities['idle-timeout'] = 120
```

Comment:
This is fine, but totally just curious: this code looks to be exactly equivalent to the old code. Why the change?